src/release/breaking-changes/ignoringsemantics-migration.md

```diff
@@ -13,7 +13,12 @@ The `ignoringSemantics` properties are removed in [`IgnorePointer`][],
 
```

Comment:
The `IgnoringSemantics` widget allows you to designate an area of the UI
where you don't want to accept pointer events, for example, when
you don't want to allow the user to enter text in a text field.

Previously, you accomplished this by setting the `ignoringSemantics` property to
true on the highest level widget you want to disable, which
removed that widget and its subtree from the semantics tree.

This behavior has changed, and the `ignoringSemantics` boolean
is deprecated. To disable an area of the UI, you now wrap
that widget in an `IgnorePointer` widget. This has the
same effect without altering the semantics tree.

This change also applies to the `AbsorbPointer` and
`SliverIgnorePointer` widgets.